gui/src/app/init/relation-utils.js

```diff
@@ -531,7 +531,7 @@ function getAvailableEndpoints(
     return relatableEndpoints;
   }
   // Filter out the existing relations
-  return relatableEndpoints.filter(relation =>
+  return relatableEndpoints && relatableEndpoints.filter(relation =>
```

Comment:
What's up with this? Should we add a test for this issue?

Reply:
This was a bug that I found while testing the events. I'd add a test, but the tests require YUI, so they're no able to be run.

Reply:
Ok that's fine you can leve it as is